src/library.js

```diff
@@ -760,6 +760,7 @@ LibraryManager.library = {
 #endif
   },
 
+#if !WASM_BACKEND
   // This object can be modified by the user during startup, which affects
   // the initial values of the environment accessible by getenv. (This works
   // in both fastcomp and upstream.
```

Comment:
the `ENV` object (line right after this one) is used from `library_wasi.js`, so I think we need that at least? We could just add `$ENV: {}` to that file perhaps.

Reply:
oops, you are correct. fixed.